server/services/omnichannel-voip/service.ts

```diff
@@ -17,12 +24,71 @@ export class OmnichannelVoipService extends ServiceClass implements IOmnichannel
 
 	private users: UsersRaw;
 
+	private voipRoom: VoipRoomsRaw;
+
 	constructor(db: Db) {
 		super();
 		this.users = new UsersRaw(db.collection('users'));
+		this.voipRoom = new VoipRoomsRaw(db.collection('rocketchat_room'));
 		this.logger = new Logger('OmnichannelVoipService');
 	}
 
+	private normalizeAgent(agentId: string): any {
```

Comment:
Do not use `any`

Reply:
Changed the return type to 'Record<string, unknown> | undefined'